src/tests/ftest/util/daos_core_base.py

```diff
@@ -103,13 +113,18 @@ def run_subtest(self):
         dmg = self.get_dmg_command()
         dmg_config_file = dmg.yaml.filename
 
+        # for 'daos_test -F' increase the number of client processes to 16
+        if subtest == "F":
+            num_clients = 16
```

Comment:
This looks like test configuration. Does it make sense to set this value in daos_core_test.yaml?